jsonrpclib/jsonclass.py

```diff
@@ -1,30 +1,16 @@
-import types
 import inspect
 import re
 
 from jsonrpclib import config
 
-iter_types = [
-    types.DictType,
-    types.ListType,
-    types.TupleType
-]
-
-string_types = [
-    types.StringType,
-    types.UnicodeType
-]
-
-numeric_types = [
-    types.IntType,
-    types.LongType,
-    types.FloatType
-]
-
-value_types = [
-    types.BooleanType,
-    types.NoneType
-]
+iter_types = (dict, list, tuple)
+value_types = (bool, )
+try:
+    string_types = (str, unicode)
+    numeric_types = (int, long, float)
+except NameError:
+    string_types = (str, )
+    numeric_types = (int, float)
```

Comment:
Same thought re: basestring, etc. here, and there may be something in the `six` library to help with iterable types.